docs/azure/integrations-overview.md

```diff
@@ -0,0 +1,360 @@
+- [📦 Azure.Provisioning.ApplicationInsights](https://www.nuget.org/packages/Azure.Provisioning.ApplicationInsights)
+- [📦 Azure.Provisioning.CognitiveServices](https://www.nuget.org/packages/Azure.Provisioning.CognitiveServices)
+- [📦 Azure.Provisioning.Communication](https://www.nuget.org/packages/Azure.Provisioning.Communication)
+- [📦 Azure.Provisioning.ContainerRegistry](https://www.nuget.org/packages/Azure.Provisioning.ContainerRegistry)
+- [📦 Azure.Provisioning.ContainerService](https://www.nuget.org/packages/Azure.Provisioning.ContainerService)
+- [📦 Azure.Provisioning.CosmosDB](https://www.nuget.org/packages/Azure.Provisioning.CosmosDB)
+- [📦 Azure.Provisioning.Deployment](https://www.nuget.org/packages/Azure.Provisioning.Deployment)
+- [📦 Azure.Provisioning.EventGrid](https://www.nuget.org/packages/Azure.Provisioning.EventGrid)
+- [📦 Azure.Provisioning.EventHubs](https://www.nuget.org/packages/Azure.Provisioning.EventHubs)
+- [📦 Azure.Provisioning.KeyVault](https://www.nuget.org/packages/Azure.Provisioning.KeyVault)
+- [📦 Azure.Provisioning.Kubernetes](https://www.nuget.org/packages/Azure.Provisioning.Kubernetes)
+- [📦 Azure.Provisioning.KubernetesConfiguration](https://www.nuget.org/packages/Azure.Provisioning.KubernetesConfiguration)
+- [📦 Azure.Provisioning.OperationalInsights](https://www.nuget.org/packages/Azure.Provisioning.OperationalInsights)
+- [📦 Azure.Provisioning.PostgreSql](https://www.nuget.org/packages/Azure.Provisioning.PostgreSql)
+- [📦 Azure.Provisioning.Redis](https://www.nuget.org/packages/Azure.Provisioning.Redis)
+- [📦 Azure.Provisioning.Search](https://www.nuget.org/packages/Azure.Provisioning.Search)
+- [📦 Azure.Provisioning.ServiceBus](https://www.nuget.org/packages/Azure.Provisioning.ServiceBus)
+- [📦 Azure.Provisioning.SignalR](https://www.nuget.org/packages/Azure.Provisioning.SignalR)
+- [📦 Azure.Provisioning.Sql](https://www.nuget.org/packages/Azure.Provisioning.Sql)
+- [📦 Azure.Provisioning.Storage](https://www.nuget.org/packages/Azure.Provisioning.Storage)
+- [📦 Azure.Provisioning.WebPubSub](https://www.nuget.org/packages/Azure.Provisioning.WebPubSub)
+- [📦 Azure.Provisioning](https://www.nuget.org/packages/Azure.Provisioning)
+
+> [!TIP]
+> You don't need to install these packages manually in your app host projects, as they're transitive dependencies of the corresponding .NET Aspire Azure hosting integrations your app host references.
+
+### Azure provisioning defaults
+
+The Azure provisioning libraries provide default values for Azure resources. These defaults are only a starting point and aren't intended for all use cases. You can customize these defaults in multiple ways to meet your specific requirements. For more information, see [Azure provisioning customization](#azure-provisioning-customization) and [Use custom Bicep templates](../deployment/azure/custom-bicep-templates.md).
+
+#### Azure role assignment defaults
```

Comment:
I think we should delete this.

Reply:
I thought we wanted to document the default role assignments?

Reply:
In each integration not in the overview

Reply:
Why should we delete it? It's it valuable for devs to know what role assignments are used?

Note that it will need to be updated when we redo the role assignments stuff in the next release.

Reply:
We don't need detailed information about each integration on the overview page. The point of this page is to explain all of the *shared* ideas and concepts for azure integrations, not a place to roll up all of the details of each one.

Reply:
Addressed